apps/E2E/src/LinkExperimental/specs/ExperimentalLink.spec.win.ts

```diff
@@ -21,6 +21,41 @@ describe('Link Testing Initialization', function () {
   });
 });
 
+describe('Link Testing Functionality', function () {
+  /* Scrolls and waits for the Button to be visible on the Test Page */
+  beforeEach(async () => {
+    await ExperimentalLinkPageObject.scrollToTestElement();
+  });
+
+  it('Validate OnPress() callback was fired on a click', async () => {
+    await ExperimentalLinkPageObject.click(ExperimentalLinkPageObject._secondaryComponent);
+    await expect(await ExperimentalLinkPageObject.didOnClickCallbackFire()).toBeTruthy(
+      `The link failed to fire an onClick callback with a mouse click.`,
```

Comment:
Let's add an errorMsg parameter to the `didOnClickCallbackFire()` function and pass the error message into that. If this function fails, it will report the error message in the `waitUntil()` method since that executed before this `expect()`.

*I believe*

@lawrencewin Can you confirm my logic is correct?